docs/source/pages/tutorials/index.rst

```diff
@@ -275,3 +275,121 @@ dependency is manually uploaded to Maven Central and does not meet the security
 You can use this policy in your GitHub Actions to prevent a deployment or fail a CI test during the
 development. Alternatively, you can treat the result as a warning and manually investigate the
 dependencies to make sure they are secure and can be trusted.
+
+-----------------------------------------------------------------
+More Accurate Analysis via Mappings of Artifacts to Exact Commits
+-----------------------------------------------------------------
+
+This tutorial demonstrates how Macaron can be used to determine the differences between one or more states of the single open source repository that produced one or more related artifacts. In this way, we show how a developer can be potentially mislead by supply chain security information that has been created for the current state of an artifact's source repository, rather than the version of the artifact they are actually using.
+
+The problem of mapping artifacts to the source code that built them is a challenging one, as most artifacts, even open source ones, do not provide a direct URL to their related repository. Services exist to make up for this lack, including Google's `Open Source Insights <https://deps.dev/>`_ tool that is in use by Macaron itself for this exact reason. However, without taking further steps, analysis of these repositories will reflect only the current state at the time of execution. One example of this is OpenSSF Scorecard, an automated tool that performs a number of a software security checks on a given project. These projects are typically provided in the form of a repository's public URL, which will be examined at its current state.
```

Comment:
It would be good to mention provenances as an alternative source of information to identify the commits, but unfortunately that requires adoption by open-source projects.